debezium-core/src/test/java/io/debezium/kafka/KafkaClusterTest.java

```diff
@@ -195,6 +198,32 @@ public void shouldStartClusterAndAllowAsynchronousProductionAndAutomaticConsumer
         assertThat(messagesRead.get()).isEqualTo(numMessages);
     }
 
+    @Test
+    public void shouldSetClusterConfigProperty() throws Exception {
+        Properties config = new Properties();
+        config.put("foo", "bar");
+        KafkaCluster kafkaCluster = new KafkaCluster().withKafkaConfiguration(config);
+
+        Field kafkaConfigField = KafkaCluster.class.getDeclaredField("kafkaConfig");
+        kafkaConfigField.setAccessible(true);
+        Properties kafkaConfig = (Properties) kafkaConfigField.get(kafkaCluster);
+        assertThat(kafkaConfig).hasSize(1);
+    }
+
+    @Test
+    public void shouldSetServerConfigProperty() throws Exception {
+        Properties config = new Properties();
+        config.put("foo", "bar");
+        KafkaCluster kafkaCluster = new KafkaCluster().withKafkaConfiguration(config).addBrokers(1);
+
+        Field kafkaServersField = KafkaCluster.class.getDeclaredField("kafkaServers");
+        kafkaServersField.setAccessible(true);
+        ConcurrentMap<Integer, KafkaServer> kafkaServers = (ConcurrentMap<Integer, KafkaServer>) kafkaServersField.get(kafkaCluster);
+        Properties serverConfig = kafkaServers.values().iterator().next().config();
+        assertThat(serverConfig.get("foo")).isEqualTo("bar");
+    }
```

Comment:
Very minor nit: one newline is enough :)

Reply:
https://issues.jboss.org/browse/DBZ-323 - is it the issue? If use let's re-use it.